conf/variant/intel/README

```diff
@@ -0,0 +1 @@
+Variant configuration for building PELUX for the Intel NUC without Qt+QtAuto
```

Comment:
Write headless or something instead of without Qt+QtAuto. Otherwise it wont scale when adding additional variants

Reply:
Non-graphical is used to distinguish this in a project I'm in. "Headless" means very different things to people it turns out :)